themes/bootstrap5/scss/bootstrap.scss

```diff
@@ -483,9 +474,14 @@ ul.list-group {
   min-width: 200px;
 }
 
-.logoutOptions.with-dropdown a { padding-right: .5rem; }
+#account-home-nav.with-dropdown a {
+  padding-right: calc(var(--bs-navbar-nav-link-padding-x) / 2);
+}
 #login-dropdown {
   a {
+    // These with the `.logoutOptions.with-dropdown a` above reduce the gap between the text and dropdown button:
```

Comment:
This comment appears to refer to a line that has been removed.

Reply:
Fixed.